src/Modules/ConfigOptions.lua

```diff
@@ -184,7 +184,7 @@ return {
 		modList:NewMod("Multiplier:FrostShieldStage", "BASE", val, "Config")
 	end },
 	{ label = "Greater Harbinger of Time:", ifSkill =  "Summon Greater Harbinger of Time" },
-	{ var = "greaterHarbingerOfTimeSlipstream", type = "check", label = "Is Slipstream active?:", ifSkill =  "Summon Greater Harbinger of Time", tooltip = "Greater Harbinger of Time Slipstream buff grants:\n20% increased Action Speed\nBuff affects the player and allies\nBuff has a base duration of 8s with a 10s Cooldown", apply = function(val, modList, enemyModList)
+	{ var = "greaterHarbingerOfTimeSlipstream", type = "check", label = "Is Slipstream active?:", ifSkill =  "Summon Greater Harbinger of Time", tooltip = "Greater Harbinger of Time Slipstream buff grants:\n10% increased Action Speed\nBuff affects the player and allies\nBuff has a base duration of 8s with a 10s Cooldown", apply = function(val, modList, enemyModList)
 		modList:NewMod("Condition:GreaterHarbingerOfTime", "FLAG", true, "Config")
 	end },
 	{ label = "Harbinger of Time:", ifSkill =  "Summon Harbinger of Time" },
```

Comment:
Could you also change the default harbinger of time slipstream buff to 10% ? 
![image](https://user-images.githubusercontent.com/17045310/128676460-880a6b6c-c0d2-410a-b095-e1454f62985d.png)